packages/wallet/src/persistence/pouchdbStores/pouchdbWalletStores.ts

```diff
@@ -25,6 +26,7 @@ export class PouchdbTimeSettingsStore extends PouchdbDocumentStore<TimeSettings[
 export class PouchdbAssetsStore extends PouchdbDocumentStore<Assets> {}
 export class PouchdbAddressesStore extends PouchdbDocumentStore<GroupedAddress[]> {}
 export class PouchdbInFlightTransactionsStore extends PouchdbDocumentStore<Cardano.NewTxAlonzo[]> {}
+export class PouchdbConfirmedTransactionsStore extends PouchdbDocumentStore<NewTxAlonzoWithSlot[]> {}
```

Comment:
```suggestion
export class PouchdbVolatileTransactionsStore extends PouchdbDocumentStore<NewTxAlonzoWithSlot[]> {}
```

Reply:
Done in: [ddb6af8](https://github.com/input-output-hk/cardano-js-sdk/pull/351/commits/ddb6af800a591149f87a2668b4da04d0470a29c7)